web/app.js

```diff
@@ -1385,9 +1399,11 @@ const PDFViewerApplication = {
         });
     });
 
-    pagesPromise.then(() => {
-      this._initializeAutoPrint(pdfDocument, openActionPromise);
-    });
+    if (!AppOptions.get("enableScripting")) {
+      pagesPromise.then(() => {
+        this._initializeAutoPrint(pdfDocument, openActionPromise);
+      });
+    }
```

Comment:
This doesn't look correct, since that will also end up ignoring the openAction-data.

Please revert these changes, and instead change https://github.com/mozilla/pdf.js/blob/d784af3f38d5604b653c06fb40a6fe6a021d45d3/web/app.js#L1592 to
```javascript
    if (javascript && !AppOptions.get("enableScripting")) {
```

---

Also, something I just realized, if you're now relying on actually executing JavaScript to *trigger printing on load* you probably want to change https://github.com/mozilla/pdf.js/blob/d784af3f38d5604b653c06fb40a6fe6a021d45d3/web/app.js#L1480 to 
```javascript
  this.pdfViewer.pagesPromise.then(() => {
    this.triggerPrinting();
  }
```
since otherwise there's a risk that it will just be ignored.